  INFO root [2018-11-08 22:30:36,635] - 
  INFO root [2018-11-08 22:30:36,656] - -----------------------------------------------------------------------------------------


  INFO root [2018-11-08 22:30:38,525] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-11-08 22:30:38,526] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-11-08 22:30:38,527] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-11-08 22:30:39,186] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20013 )
  INFO root [2018-11-08 22:30:39,281] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20013, SOMAXCONN=128.
  INFO root [2018-11-08 22:30:39,286] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:49842 )
  INFO root [2018-11-08 22:30:39,287] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:49842, SOMAXCONN=128.
  INFO root [2018-11-08 22:30:39,288] - server-configs:
  INFO root [2018-11-08 22:30:39,288] - 	internalAddr : 192.168.0.107:49842
  INFO root [2018-11-08 22:30:39,289] - 	externalAddr : 192.168.0.107:20013
  INFO root [2018-11-08 22:30:39,289] - 	externalCustomAddr : c.55908.3m.dkys.org
  INFO root [2018-11-08 22:30:39,289] - 	componentID : 9000
  INFO root [2018-11-08 22:30:41,252] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 26874. PID: 5504 ----
  INFO root [2018-11-08 22:30:41,253] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-11-08 22:30:41,509] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-11-08 22:30:42,040] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-11-08 22:30:49,288] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-11-08 22:30:49,493] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-11-08 22:30:49,504] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-11-08 22:30:50,598] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-11-08 22:30:50,741] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-11-08 22:30:50,812] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-11-08 22:30:50,833] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-11-08 22:30:53,685] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-11-08 22:30:55,873] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-11-08 22:30:55,874] - Script::install(): KBExtra is successfully!
S_INFO root [2018-11-08 22:30:56,904] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-11-08 22:30:56,905] - TelnetServer server is running on port 31000
  WARN root [2018-11-08 22:30:56,906] - e:\kbengine\kbe\src\server\loginapp\loginapp.cpp:153: Profile scriptCall took 0.21 seconds
  INFO root [2018-11-08 22:30:58,820] - ---- loginapp is running ----
 DEBUG root [2018-11-08 22:30:58,940] - Components::process(): Request for the process of identity...
  WARN root [2018-11-08 22:31:05,468] - e:\kbengine\kbe\src\lib\network\event_dispatcher.cpp:153: Profile callSystemTimers took 0.10 seconds
  INFO root [2018-11-08 22:31:06,478] - Components::findComponents: find logger(1)...
  INFO root [2018-11-08 22:31:06,569] - Components::findComponents: found logger, addr:192.168.0.107:49838
  INFO root [2018-11-08 22:31:06,569] - Components::addComponent: reset orderLog, uid=26874!
  INFO root [2018-11-08 22:31:06,570] - Components::addComponent[logger], uid=26874, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-11-08 22:31:06,669] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-11-08 22:31:06,774] - Components::findComponents: found dbmgr, addr:192.168.0.107:49840
  INFO root [2018-11-08 22:31:06,775] - Components::addComponent: reset orderLog, uid=26874!
  INFO root [2018-11-08 22:31:06,775] - Components::addComponent[dbmgr], uid=26874, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-11-08 22:31:06,776] - Components::findComponents: found dbmgr, addr:192.168.0.107:49840
  WARN root [2018-11-08 22:31:07,777] - Forwarding logs to logger[192.168.0.107:49838/0/0/0]...
 ERROR root [2018-11-08 23:07:12,927] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-11-08 23:07:12,930] - The following logs sent to logger failed:
  WARN root [2018-11-08 23:07:12,931] - ==> [2018-11-08 23:07:12 926] TCPPacketReceiver::processPendingEvents(192.168.0.107:49838): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-11-08 23:07:12,931] - ==> [2018-11-08 23:07:12 927] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-11-08 23:07:12,970] - TCPPacketReceiver::processPendingEvents(192.168.0.107:49840): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-11-08 23:07:12,971] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-11-08 23:07:12,974] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-11-08 23:07:12,974] - onLoginAppShutDown()
  INFO root [2018-11-08 23:07:12,974] - Shutdowner::onShutdown: shutting down
  INFO root [2018-11-08 23:07:12,974] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-11-08 23:07:12,975] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-11-08 23:07:13,018] - Script::uninstall(): is successfully!
 DEBUG root [2018-11-08 23:07:13,018] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-11-08 23:07:14,096] - 
  INFO root [2018-11-08 23:07:14,096] - -----------------------------------------------------------------------------------------


  INFO root [2018-11-08 23:07:14,096] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-11-08 23:07:14,097] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-11-08 23:07:14,097] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-11-08 23:07:14,102] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20013 )
  INFO root [2018-11-08 23:07:14,102] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20013, SOMAXCONN=128.
  INFO root [2018-11-08 23:07:14,104] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:50179 )
  INFO root [2018-11-08 23:07:14,104] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:50179, SOMAXCONN=128.
  INFO root [2018-11-08 23:07:14,104] - server-configs:
  INFO root [2018-11-08 23:07:14,104] - 	internalAddr : 192.168.0.107:50179
  INFO root [2018-11-08 23:07:14,104] - 	externalAddr : 192.168.0.107:20013
  INFO root [2018-11-08 23:07:14,104] - 	externalCustomAddr : c.55908.3m.dkys.org
  INFO root [2018-11-08 23:07:14,104] - 	componentID : 9000
  INFO root [2018-11-08 23:07:15,782] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 1382. PID: 8000 ----
  INFO root [2018-11-08 23:07:15,783] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-11-08 23:07:15,783] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-11-08 23:07:15,783] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-11-08 23:07:15,905] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-11-08 23:07:15,905] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-11-08 23:07:15,906] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-11-08 23:07:15,908] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-11-08 23:07:15,909] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-11-08 23:07:15,909] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-11-08 23:07:15,909] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-11-08 23:07:15,935] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-11-08 23:07:15,962] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-11-08 23:07:15,963] - Script::install(): KBExtra is successfully!
S_INFO root [2018-11-08 23:07:15,964] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-11-08 23:07:15,964] - TelnetServer server is running on port 31000
  INFO root [2018-11-08 23:07:16,229] - ---- loginapp is running ----
 DEBUG root [2018-11-08 23:07:16,229] - Components::process(): Request for the process of identity...
  INFO root [2018-11-08 23:07:23,345] - Components::findComponents: find logger(1)...
  INFO root [2018-11-08 23:07:23,410] - Components::findComponents: found logger, addr:192.168.0.107:50175
  INFO root [2018-11-08 23:07:23,410] - Components::addComponent: reset orderLog, uid=1382!
  INFO root [2018-11-08 23:07:23,411] - Components::addComponent[logger], uid=1382, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-11-08 23:07:23,413] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-11-08 23:07:23,517] - Components::findComponents: found dbmgr, addr:192.168.0.107:50177
  INFO root [2018-11-08 23:07:23,517] - Components::addComponent: reset orderLog, uid=1382!
  INFO root [2018-11-08 23:07:23,518] - Components::addComponent[dbmgr], uid=1382, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-11-08 23:07:23,518] - Components::findComponents: found dbmgr, addr:192.168.0.107:50177
  WARN root [2018-11-08 23:07:24,518] - Forwarding logs to logger[192.168.0.107:50175/0/0/0]...
